voiceapp/views.py

```diff
@@ -0,0 +1,57 @@
+from django.shortcuts import render
+from django.http import JsonResponse
+from transformers import pipeline
+import json
+from django.views.decorators.csrf import csrf_exempt
+
+# Load intent classification model globally
+classifier = pipeline("zero-shot-classification", model="facebook/bart-large-mnli")
```

Comment:
should this be added to the requirements.txt (lib), or is the model loaded from the web?
Asking to make sure no errors show up in other devices while trying to run the project.